sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -50,9 +51,13 @@ public ClientLogger(Class<?> clazz) {
      * Retrieves a logger for the passed class name using the {@link LoggerFactory}.
      *
      * @param className Class name creating the logger.
+     * @throws RuntimeException it is an error.
      */
     public ClientLogger(String className) {
         logger = LoggerFactory.getLogger(className);
+        System.setProperty("java.util.logging.config.file",
+            ClientLogger.class.getClassLoader().getResource("logging.properties").getPath());
+        defaultLogger = java.util.logging.Logger.getLogger(className);
```

Comment:
Don't overwrite a system property. Use another means to configure the logger. Also, don't create a default logger unless necessary - you're paying the price here to create two loggers.

Reply:
We can initial it into log method. Try to figure out if we can set the logLevel during runtime.